src/promptflow/tests/executor/e2etests/test_executor_validation.py

```diff
@@ -243,20 +242,10 @@ def test_batch_run_input_type_invalid(
         output_dir = Path(mkdtemp())
         batch_results = batch_engine.run(input_dirs, inputs_mapping, output_dir)
 
-        if (
-            (sys.version_info.major == 3)
-            and (sys.version_info.minor >= 11)
-            and ((sys.platform == "linux") or (sys.platform == "darwin"))
-        ):
-            # Python >= 3.11 has a different error message on linux and macos
-            error_message_compare = error_message.replace("int", "ValueType.INT")
-            assert error_message_compare in str(
-                batch_result.line_results[0].run_info.error
-            ), f"Expected message {error_message_compare} but got {str(batch_result.line_results[0].run_info.error)}"
-        else:
-            assert error_message in str(
-                batch_results.error_summary.error_list[0].error
-            ), f"Expected message {error_message} but got {str(batch_results.error_summary.error_list[0].error)}"
+        assert error_message in str(
```

Comment:
![image](https://github.com/microsoft/promptflow/assets/38847871/cceaac5d-2b35-44f0-9b6e-5f134b74e831)

according to matrix test, there are some other tests fail exclusively in Python 3.11, and I guess it's also related to Enum? maybe also take a look on this?